frontend/src/components/App.js

```diff
@@ -149,6 +152,7 @@ export const mapStateToProps = state => ({
 export const actionCreators = {logOut, get};
 
 const App = ({userID, usersByID, logOut, get}) => {
+  const env = FMS_ENV
```

Comment:
```suggestion
  /* global FMS_ENV */
  const env = FMS_ENV
```

Adding a 'global'  comment tells eslint that FMS_ENV is defined in the global environment.